### ISSUE TITLE
None values in lightcurves for snn

### ISSUE THREAD

**Issue by anaismoller** [astrolabsoftware/fink-science#56]
SNN was not designed to handle None values in light-curves. A simple pdf = pdf[~ pdf.FLUXCAL.isna()] in line 120 in snn/processor.py should deal with that

**Comment by anaismoller** [comment]
Is there an action you need from me?

**Comment by JulienPeloton** [comment]
Nope! I guess you received a notification because I was doing triage on issues ;-)
